### ISSUE TITLE
Does MPRIS expose Genre

### ISSUE THREAD

**Issue by bonelifer** [InputUsername/rescrobbled#70]
Does MPRIS expose Genre? If so it'd be nice to be able to filter on genre,

**Comment by InputUsername** [comment]
It does actually, and it seems reasonable to allow filtering on it. Should be easy enough to add! ~~(famous last words)~~

**Comment by bonelifer** [comment]
Thanks or looking into it. Would be nice for filtering Podcasts.

**Comment by bonelifer** [comment]
Yes I would. I download my podcasts with GPodder and will be using MPD with a mpd MPRIS client.

**Comment by bonelifer** [comment]
Sure. I need to setup one of the MPRIS clients first as I have been deep diving into github/gist and google for interesting MPD related scripts.

**Comment by InputUsername** [comment]
@bonelifer thanks. Genre filtering is implemented on the [`filter-genre`](https://github.com/InputUsername/rescrobbled/tree/filter-genre/) branch. I can also send over a binary if needed.

Apologies for the late reply.

**Comment by bonelifer** [comment]
Going to test later. But have this problem with the systemd file:
```
william@william:/etc/systemd/system$ sudo systemctl status rescrobbled.service 
● rescrobbled.service - An MPRIS scrobbler
     Loaded: loaded (/etc/systemd/system/rescrobbled.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2022-06-14 11:02:31 CDT; 9min ago
       Docs: https://github.com/InputUsername/rescrobbled
   Main PID: 34145 (code=exited, status=1/FAILURE)

Jun 14 11:02:31 william systemd[1]: Started An MPRIS scrobbler.
Jun 14 11:02:31 william rescrobbled[34145]: Authenticated with Last.fm successfully!
Jun 14 11:02:31 william rescrobbled[34145]: Error: Failed to connect to D-Bus
Jun 14 11:02:31 william rescrobbled[34145]: Caused by:
Jun 14 11:02:31 william rescrobbled[34145]:     D-Bus call failed: "Unable to autolaunch a dbus-daemon without a $DISPLAY for X11"
Jun 14 11:02:31 william systemd[1]: rescrobbled.service: Main process exited, code=exited, status=1/FAILURE
Jun 14 11:02:31 william systemd[1]: rescrobbled.service: Failed with result 'exit-code'.

```

**Comment by bonelifer** [comment]
No it gets better, tried re-enabling it again, now this status error:
`william@william:/etc/systemd/system$ sudo systemctl status rescrobbled.service 
● rescrobbled.service - An MPRIS scrobbler
     Loaded: loaded (/etc/systemd/system/rescrobbled.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2022-06-14 11:20:25 CDT; 2min 15s ago
       Docs: https://github.com/InputUsername/rescrobbled
   Main PID: 36685 (code=exited, status=203/EXEC)

Jun 14 11:20:25 william systemd[1]: Started An MPRIS scrobbler.
Jun 14 11:20:25 william systemd[36685]: rescrobbled.service: Failed to execute command: No such file or directory
Jun 14 11:20:25 william systemd[36685]: rescrobbled.service: Failed at step EXEC spawning /root/.cargo/bin/rescrobbled: No such file or directory
Jun 14 11:20:25 william systemd[1]: rescrobbled.service: Main process exited, code=exited, status=203/EXEC
Jun 14 11:20:25 william systemd[1]: rescrobbled.service: Failed with result 'exit-code'.
`

**Comment by bonelifer** [comment]
Forget those errors. Restarted and all is well. Such a weird day. Will report back later.

**Comment by bonelifer** [comment]
Ok, tested. Works.

**Comment by InputUsername** [comment]
Perfect, thank you! I'll release this in the next version, hopefully soon.

**Comment by InputUsername** [comment]
@bonelifer I've just released version 0.6.0 which includes this functionality!
